nx-X11/programs/Xserver/hw/nxagent/man/nxagent.1

```diff
@@ -372,6 +372,12 @@ force use of NX protocol messages assuming communication through nxproxy
 .B \-nxrealwindowprop
 set property NX_REAL_WINDOW for each X11 client inside NX Agent, providing the window XID of the corresponding window object on the X server that NX Agent runs on
 .TP 8
+.B \-reportwids
+explicitly tell NX Agent to report its externally exposed X11 window IDs to the session log (in machine readable form), so that external parsers can obtain that information from there
+.TP 8
+.B \-reportprivatewids
+explicitly tell NX Agent to report X11 window IDs of internally created window objects to the session log (in machine readable form), so that external parsers can obtain that information from there; this creates a lot of output and may affect performance
+.TP 8
```

Comment:
So is this really a debugging option? Or is is needed for Telekinesis to work?

Generally: why has this to be in the session log? Shouldn't we rather user a separate logfile with only machine readable stuff like this?